### PR TITLE
docs: Some Wireguard improvements

### PR DESCRIPTION
**Pull request by brb**
See commit msgs.

### CODE REVIEW

**Review by gandro — APPROVED** [review]
Nice!

File: Documentation/gettingstarted/encryption-wireguard.rst [hunk 1]
```diff
@@ -73,18 +73,28 @@ Run a ``bash`` shell in one of the Cilium pods with
 ``kubectl -n kube-system exec -ti ds/cilium -- bash`` and execute the following
 commands:
 
-1. Install tcpdump
+1. Check that Wireguard has been enabled (number of peers should correspond to
+   a number of nodes subtracted by one):
+
+.. code:: shell-session
```
Comment (qmonnet) [inline Documentation/gettingstarted/encryption-wireguard.rst]:
```suggestion
.. code-block:: shell-session
```

**Review by qmonnet — APPROVED** [review]
Looks good!

**Comment by brb** [comment]
@gandro @qmonnet Thanks for the review. Marking as ready-to-merge.

**Comment by qmonnet** [comment]
Based on the `release-blocker/1.10`, I'm tagging for 1.10 backports.

**Pull request merged**